src/backend/mod.rs

```diff
@@ -21,6 +21,12 @@
 //! `32bit` since identifiers can't start with letters, and the backends
 //! do use `u32`/`u64`, so this seems like a least-bad option.
 
+#[cfg(not(any(feature = "u32_backend", feature = "u64_backend", feature = "avx2_backend")))]
+compile_error!(
+    "no curve25519-dalek backend cargo feature enabled! \
+     please enable one of: u32_backend, u64_backend, avx2_backend"
+);
```

Comment:
Threw this in too... it results in:

```
   Compiling curve25519-dalek v0.18.0 (file:///Users/bascule/src/curve25519-dalek)
error: no curve25519-dalek backend cargo feature enabled! please enable one of: u32_backend, u64_backend, avx2_backend
  --> src/backend/mod.rs:25:1
   |
25 | / compile_error!(
26 | |     "no curve25519-dalek backend cargo feature enabled! \
27 | |      please enable one of: u32_backend, u64_backend, avx2_backend"
28 | | );
   | |__^

error: aborting due to previous error
```

...if no backend is selected.

Reply:
Nice!!